### DISCUSSION TITLE
The mobile version cannot find the forks entry

### DISCUSSION THREAD

**Discussion by liushichuan** [community/community#101787] [Mobile]
Mobile version currently can not fork an item, not supported or not obvious? I can't find the operating entrance. Hope the mobile version can support the forks operation, and the operation entrance can be more prominent.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, 版本 1.145.0 (120653769)
Device: iPhone14,7 (iOS 17.3)
</details>

**Comment by eliperkins** [comment]
Hey @liushichuan, thanks for this feedback. You're right, there is ability to fork a repo within GitHub Mobile today. I'd love to get some more info from you:

- After forking a repo on GitHub Mobile, what would be the next action you'd want to take with that repo?
- Without forking in GitHub Mobile today, how do you work around this problem?

I've also added your +1 for forking repos on GitHub Mobile to our internal tracking issue for this. Thanks for helping us improve GitHub Mobile!

**Comment by liushichuan** [comment]
Today, git hub is not just a community of developers, but also includes learning resources, such as an index of learning routes in a field. Usually I get some suggestions from other social media or Google News, and if I think it's good, I'll fork it down to make sure the information is not lost.For a library of a class, the forks are similar to the snaps, preventing you from learning while the contents of the library continue to change, such as the selected reference is best still.
The next step after the forks are done, if it is code, I will access it again through the pc when there are working conditions, if it is a guide I may continue to read repeatedly on the phone for many days.
Mobile web version can be clickable to a branch of the Forks Store. It is not clear why the current client version and the web version do not work together, which has actually caused confusion and additional trial and learning costs for users. The problem with the mobile version is that the login password may expire and need to be re-entered, and accessing the repository again is not direct enough. Many browser tabs are often intertwined with repositories, which is why I chose to use githubbs for forks instead of keeping repositories as browser bookmarks.

There are so many Github users.Developers are difficult to enumerate the use of all people's habits, first maintain consistency is particularly important, the current arbitrary function has undergone the test of time, innovation should not be overturned again, otherwise sometimes can not attract users or even play the role of exclusion.

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: Eli Perkins ***@***.***>
发送时间: Thursday, January 25, 2024 5:12:51 AM
收件人: community/community ***@***.***>
抄送: liusc ***@***.***>; Mention ***@***.***>
主题: Re: [community/community] The mobile version cannot find the forks entry (Discussion #101787)


Hey @liushichuan<https://github.com/liushichuan>, thanks for this feedback. You're right, there is ability to fork a repo within GitHub Mobile today. I'd love to get some more info from you:

  *   After forking a repo on GitHub Mobile, what would be the next action you'd want to take with that repo?
  *   Without forking in GitHub Mobile today, how do you work around this problem?

I've also added your +1 for forking repos on GitHub Mobile to our internal tracking issue for this. Thanks for helping us improve GitHub Mobile!

―

**Comment by jsoref** [comment]
#3333 asked for this. 

https://twitter.com/github/status/1866883185742622746 claims the feature exists

**Reply by iskae** [reply]
Hey @jsoref 👋 

Yes, that is correct 🎉 We are currently rolling out the feature gradually to production 🙇 Please stay tuned 😊 

Cheers!
